base/Angle.hpp

```diff
@@ -106,6 +107,20 @@ class Angle
         return result;
     }
 
+    /** Use this method to get the minimum acceptable angle
```

Comment:
The documentation is wrong. It is the maximum angle returned by getRad(). If you use fromRad, you are allowed to give any value, and it will be normalized to the range from min <-> max.